### ISSUE TITLE
Support dotenv files

### ISSUE THREAD

**Issue by fromz** [yext/edward#151]
Has anyone put any thought into supporting dotenv files for environment variable management natively? I'm happy to do the dev work if we agree it's a good idea

**Comment by theothertomelliott** [comment]
Interesting idea! Do you have any thoughts on particular use cases for this feature? I think it could be useful, but there's potential for clashes or confusing behavior when launching services that already use dotenv internally.

**Comment by fromz** [comment]
Yeah, so we as a dev team heavily rely on edward for our services. Some of us have different environment variable configurations, and it would be good to be able to easily maintain that between machines without having to manage dotenv manually. I think if we made the configuration very distinct between dotenv and the existing "env" configuration, it should be fairly clear, especially if we spell out the order of precedence - e.g. shell set environment variables first, then edward "env" second, then dotenv last, or some such.

**Comment by theothertomelliott** [comment]
Sounds like a decent order of precedence. Would you imagine dotenv being an "always on" feature, or something you'd enable for a specific Edward config, group or service?

Are the different variable configurations on your team personal settings per developer (perhaps being manually set and listed in gitignore)?

**Comment by fromz** [comment]
I'd say per service initially and then maybe add it per group. What do you
think? Happy to send a pr in the next few days if you agree

**Comment by theothertomelliott** [comment]
Sounds good, look forward to seeing your PR!
We've been adding features on the develop branch and merging for versioned releases.
